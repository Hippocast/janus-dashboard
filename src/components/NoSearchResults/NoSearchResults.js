import React from 'react'

import InfoPanel from '../InfoPanel/InfoPanel'

const style = {
  fontSize: 50,
  fontWeight: 300
}

const SadFace = () => <p style={style}>&#9785;</p>

const NoSearchResults = () => <InfoPanel
  icon={SadFace()}
  text='Nothing found'
/>

export default NoSearchResults
