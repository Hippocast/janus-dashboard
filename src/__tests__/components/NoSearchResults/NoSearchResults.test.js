/* eslint-env jest */
import React from 'react'
import renderer from 'react-test-renderer'

import NoSearchResults from '../../../components/NoSearchResults/NoSearchResults'

describe('NoSearchResults component', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <NoSearchResults
        />
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
