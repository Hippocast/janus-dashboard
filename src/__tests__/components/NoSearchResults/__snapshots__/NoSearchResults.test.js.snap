// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoSearchResults component renders correctly 1`] = `
<div
  className="info-panel j-pane"
>
  <p
    style={
      Object {
        "fontSize": 50,
        "fontWeight": 300,
      }
    }
  >
    â˜¹
  </p>
  <p
    className="info-panel__text"
  >
    Nothing found
  </p>
</div>
`;
